components/cards.css

```diff
@@ -0,0 +1,213 @@
+:root {
+    --badge-background: #e0e0e03b;
+    --badge-color-green: #3dd33d;
+    --size-active: #555353;
+    --badge-background-overlay: #eaeaec99;
+    --box-shadow-card: rgba(0, 0, 0, 0.25) 0px 0.0625em 0.0625em, rgba(0, 0, 0, 0.25) 0px 0.125em 0.5em, rgba(255, 255, 255, 0.1) 0px 0px 0px 1px inset;
+}
+
+.card-badge {
+    background-color: var(--badge-background);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 18rem;
+    height: max-content;
+    padding: 0.8rem;
+    border-radius: 8px;
+}
+
+.product {
+    border-radius: 8px;
+}
+
+.prod-container {
+    position: relative;
+}
+
+.prod-container img {
+    max-width: 100%;
+    height: auto;
+    height: 100%;
+    width: 100%;
+    border-radius: 8px;
+}
+
+button {
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+.prod-content {
+    padding: 8px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 0.8rem;
+}
+
+.prod-info {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 100%;
+}
+
+.btn-badge {
+    color: var(--background-color);
+    font-size: smaller;
+    border-radius: 5px;
+    font-weight: 600;
```

Comment:
Suggestion:
You can store this font-weight as a variable in root as font-semi-bold if this value is used frequently throughout multiple files.